installer/dev/WindowsAppSDKInstall.vcxproj

```diff
@@ -127,14 +127,14 @@
     <None Include="packages.config" />
     <None Include="PropertySheet.props" />
   </ItemGroup>
-  <ItemDefinitionGroup Condition="Exists('project_reunion_definitions_override.h')">
+  <ItemDefinitionGroup Condition="Exists('windowsappsdk_definitions_override.h')">
```

Comment:
The name of this is not consistent with earlier reference to it in the .gitignore, which is "windows_apps_sdk_definitions_override.h"  In other places the default definitions is called "windowsappsdk_definitions.h" so I would prefer "windowsappsdk_definitions_override.h" for the override header, meaning this is correct but other references to windows_apps_sdk_definitions_override.h need to be changed.

Reply:
Good catch. Thanks